static/src/javascripts/projects/common/modules/commercial/ticker.js

```diff
@@ -93,17 +88,10 @@ const populateGoal = (parentElement: HTMLElement) => {
 const animate = (parentElementSelector: string) => {
     const parentElement = document.querySelector(parentElementSelector);
 
-    const tickerElementSelector =
-        total > goal ? '.js-ticker-over-goal' : '.js-ticker-under-goal';
+    const tickerElementSelector = '.js-ticker-amounts';
 
     if (parentElement && parentElement instanceof HTMLElement) {
-        const heightClass =
-            total > goal ? 'epic-ticker--short' : 'epic-ticker--tall';
-        parentElement.classList.add(heightClass);
```

Comment:
I think this `heightClass` bit might have been the only change you needed to make in this file, other than setting `allowToExceed` to `true`